### PR TITLE
RSDK-2806: Movement sensor distance values should be in meters

### PR DESCRIPTION
**Pull request by stevebriskin**

### CODE REVIEW

**Review by cheukt — APPROVED** [review]

File: proto/viam/component/movementsensor/v1/movementsensor.proto [hunk 1]
```diff
@@ -149,7 +149,7 @@ message GetAccuracyRequest {
 }
 
 message GetAccuracyResponse {
-  map<string, float> accuracy_mm = 1;
+  map<string, float> accuracy = 1;
```
Comment (penguinland) [inline proto/viam/component/movementsensor/v1/movementsensor.proto:152]: Discussion question: the old name included the units. Should the new name include units, too? Like `accuracy_m` so it's clear that it's not percentages.

**Review by penguinland — APPROVED** [review]
LGTM. 

What's the plan for communicating this to the rest of the world? Changing the field names within protobufs is a pretty major breaking change, and I imagine people who are already using the accelerometers on their rovers will need to change their code to match.

**Comment by penguinland** [comment]
(Just to be clear: I reviewed the .proto file but skipped all the .ts and .go ones. If you have manually changed anything in the other files, I won't have noticed.)

File: proto/viam/component/movementsensor/v1/movementsensor.proto (hunk 1 above)
Comment (stevebriskin) [reply proto/viam/component/movementsensor/v1/movementsensor.proto:152]: having unit in accuracy was a mistake. accuracy may refer to a variety of units. for example, current usage is limited to gps which are unitless values as far as i can tell.

Comment (penguinland) [reply proto/viam/component/movementsensor/v1/movementsensor.proto:152]: I had thought the GPS readings were accurate to within a couple meters (or a couple centimeters, if you have the RTK correction data), and these units made sense (the accuracy is measured in millimeters). You make a good point, though, that the gyroscope accuracy is likely in degrees per second, so the name of the map itself cannot describe the units of its values. Instead, the keys in the map could describe the units in their corresponding value. I do think it's important that units are associated with the data in some way.

Comment (penguinland) [reply proto/viam/component/movementsensor/v1/movementsensor.proto:152]: (but the keys are in the RDK repo, so no further changes are needed in this PR)

**Pull request merged**